### PR TITLE
PP9-10795 - Content Portal: XMP Metadata layer is being displayed

### PR DESCRIPTION
**Pull request by tc4rdpp**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services/services.extensions.ts [hunk 1]
```diff
@@ -63,6 +64,12 @@ class ShareContentDetail extends generated.ShareContentDetail {
   }
 }
 
+class SchemaDetail extends generated.SchemaDetail {
+  get filteredLayerSchemaIds(): string[] | undefined {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services/services.extensions.ts:68]: Find a better name and add jsdoc comments. e.g. nonSystemLayerSchemaIds

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts [hunk 2]
```diff
@@ -44,8 +44,8 @@ export class LayerPanelsComponent implements OnInit {
         const isVirtualContent = this.content.isVirtual();
         const schemas = this.showContentSchema && isVirtualContent ? [this.content.contentSchemaId] : [];
 
-        if (contentSchema.layerSchemaIds) {
-          schemas.push(...contentSchema.layerSchemaIds);
+        if (contentSchema.filteredLayerSchemaIds) {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts:47]: It should still be possible to show XMP / Exif layers. Maybe pass excludedSchemaIds to layer-panels component?

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services/services.extensions.ts (hunk 1 above)
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services/services.extensions.ts]: Removed the property as it is not used

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts (hunk 2 above)
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services/services.extensions.ts [hunk 3]
```diff
@@ -38,6 +37,8 @@ export const NON_VIRTUAL_CONTENT_SCHEMAS_IDS = [
   'VectorMetadata',
 ];
 
+export const SYSTEM_CONTENT_SCHEMAS_IDS = ['XmpMetadata', 'ExifMetadata'];
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services/services.extensions.ts:40]: Should be SYSTEM_LAYER_SCHEMA_IDS

Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services/services.extensions.ts]: Done

**Pull request merged**